.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Run tests on pull requests
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1, 2]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run Cypress tests
+        uses: cypress-io/github-action@v5
+        timeout-minutes: 5
+        with:
+          browser: chrome
+          start: yarn dev --host
```

Comment:
No caso o workflow falhou pq a opçao host tá sendo usada sem nenhum argumento

Reply:
Just think I found a solution.
Next listens to 0.0.0.0 by default when you run "next dev". 

Source: https://nextjs.org/docs/app/api-reference/next-cli#development
"You can also set the **hostname to be different from the default of 0.0.0.0**, this can be useful for making the application available for other devices on the network. The default hostname can be changed with -H, like so:"